deploy/docker/cp-edge/sync-routes.py

```diff
@@ -680,3 +780,5 @@ def load_pods_for_runs_with_endpoints():
                 print('Service url ({}) assigned to RunID: {}'.format(service_urls_json, run_id))
         else:
                 print('Service url was not assigned due to API errors')
+
```

Comment:
With this approach `plain` endpoints are initialised only when all dns records will be created, so users will have to wait. Can we improve this?